.github/workflows/test.yml

```diff
@@ -71,6 +71,7 @@ jobs:
           pip install "jupyterlab>=4.0.0,<5"
           pip install -e .
           jlpm
+          jlpm build
```

Comment:
Why do we need it here?

Reply:
I don't remember the reason, I'll remove it to see what happens.

Reply:
Maybe it will be fine once `@jupyter/collaborative-drive` is published?

Reply:
Ah, maybe I can help publish a placeholder version to see if it helps.

Reply:
Just published the package: https://www.npmjs.com/package/@jupyter/collaborative-drive

Reply:
It looks like a version with the previous name was also published earlier: https://www.npmjs.com/package/@jupyter/collaborativedrive

Maybe it can be removed from `npm` so it's less confusing?

Reply:
Now I'm confused, who published that? Do we use it somewhere?

Reply:
> Now I'm confused, who published that? Do we use it somewhere?

Looks like you did?

![image](https://github.com/user-attachments/assets/b132f539-bf1c-4b74-9a12-5ca68f964f3d)

Reply:
I guess you published it to see if that would help fix the CI?